src/models/chunks/YesNo.schema.ts

```diff
@@ -0,0 +1,18 @@
+import Joi from '@hapi/joi';
+
+import { YesNo } from 'app/models/chunks/YesNo';
+
+export const createSchema = (errorMessage: string): Joi.StringSchema => {
+    return Joi.string()
+        .required()
+        .valid(YesNo.yes, YesNo.no)
+        .messages({
+            'any.required': errorMessage,
+            'any.only': errorMessage,
+            'string.base': errorMessage,
+            'string.empty': errorMessage
+        })
+        .options({
+            abortEarly: true
+        })
+};
```

Comment:
Why did you call it Chunks? Doesn't mean much to me, could it be renamed?

Reply:
Cause I could not find better name. The intention is to have field level reusable schemas for things like Yes/No, penalty reference, company number and so on. How would you name it?

Reply:
What about `core`?

Reply:
Renamed to `fields` as per our conversation.